ui/app/src/components/shared/ResourceOperationStepsList.tsx

```diff
@@ -23,15 +25,33 @@ export const ResourceOperationStepsList: React.FunctionComponent<
       </Stack.Item>
       <div style={{ width: "80%" }}>
         {props.val?.map((step: OperationStep, i: number) => {
+          const [isErrorPanelOpen, setIsErrorPanelOpen] = React.useState(false);
+          const isError =
+            typeof step.message === "string" &&
+            (step.message.includes("Error:") || step.message.includes("error:"));
+
           return (
             <Stack.Item styles={stackItemStyles} key={i}>
               <div>
                 {i + 1}
                 {")"} {step.stepTitle}
               </div>
-              <div style={{ color: DefaultPalette.neutralTertiary }}>
-                {step.message}
-              </div>
+              {isError ? (
+                <>
+                  <Link onClick={() => setIsErrorPanelOpen(true)}>
+                    An error occurred, click to view the error details
```

Comment:
nit: Can we make this two sentences, or use a semi-colon please?
```suggestion
                    An error occurred; click to view the error details
```